components/CheckDocumentation/vacuum/XminHorizon.tsx

```diff
@@ -135,17 +173,22 @@ const GuidanceByBackend: React.FunctionComponent<{
                   ORDER BY greatest(age(backend_xmin), age(backend_xid)) DESC;`}
         />
       </CodeBlock>
-      <p>You can cancel it by running either of commands:</p>
+      <p>You can cancel it by running either <a href="https://www.postgresql.org/docs/current/functions-admin.html#FUNCTIONS-ADMIN-SIGNAL" target="_blank">pg_cancel_backend</a> if the transaction is running an active query:</p>
```

Comment:
We have `<PGDocsLink />` for linking to Postgres documentation; we should probably use that here.